icons/figma.svg

```diff
@@ -9,9 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M5 5.5A3.5 3.5 0 0 1 8.5 2H12v7H8.5A3.5 3.5 0 0 1 5 5.5z" />
-  <path d="M12 2h3.5a3.5 3.5 0 1 1 0 7H12V2z" />
-  <path d="M12 12.5a3.5 3.5 0 1 1 7 0 3.5 3.5 0 1 1-7 0z" />
-  <path d="M5 19.5A3.5 3.5 0 0 1 8.5 16H12v3.5a3.5 3.5 0 1 1-7 0z" />
-  <path d="M5 12.5A3.5 3.5 0 0 1 8.5 9H12v7H8.5A3.5 3.5 0 0 1 5 12.5z" />
+  <circle cx="15.5" cy="12.5" r="3.5"/>
+  <rect x="5" y="2" width="14" height="7" rx="3.5"/>
+  <path d="M12 2v17.5A3.5 3.5 0 1 1 8.5 16H12 8.5a3.5 3.5 0 0 1 0-7"/>
```

Comment:
The Figma icon still violates the guidelines (1px padding box), but will likely be removed soon anyway: #94 
```suggestion
  <circle cx="15.5" cy="12.5" r="3.5" />
  <rect x="5" y="2" width="14" height="7" rx="3.5" />
  <path d="M12 2v17.5A3.5 3.5 0 1 1 8.5 16H12 8.5a3.5 3.5 0 0 1 0-7" />
```